<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motion</class>
 <widget class="QWidget" name="Motion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Drawing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="drawPath">
        <property name="text">
         <string>Path</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawRRT">
        <property name="text">
         <string>RRT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawHistory">
        <property name="text">
         <string>Path History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawBezierTraj">
        <property name="text">
         <string>Bezier Trajectory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawBezierControl">
        <property name="text">
         <string>Bezier Control Points</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
